regression-test/suites/sql_block_rule_p0/test_sql_block_rule.groovy

```diff
@@ -170,7 +170,7 @@ suite("test_sql_block_rule") {
     """
 
     sql """
-        CREATE SQL_BLOCK_RULE if not exists test_rule_partition PROPERTIES ( "partition_num" = "1", "global" = "true",
+        CREATE SQL_BLOCK_RULE if not exists test_rule_partition PROPERTIES ( "partition_num" = "1", "global" = "false",
```

Comment:
block rule with global = false is not work

Reply:
set it global may affect other test cases